adafruit_hid/consumer_control.py

```diff
@@ -35,23 +35,27 @@
 # pylint: disable=wrong-import-position
 import struct
 import time
-import usb_hid
 
 class ConsumerControl:
     """Send ConsumerControl code reports, used by multimedia keyboards, remote controls, etc.
-
-    *New in CircuitPython 3.0.*
     """
 
-    def __init__(self):
-        """Create a ConsumerControl object that will send Consumer Control Device HID reports."""
-        self.hid_consumer = None
-        for device in usb_hid.devices:
-            if device.usage_page == 0x0C and device.usage == 0x01:
-                self.hid_consumer = device
-                break
-        if not self.hid_consumer:
-            raise IOError("Could not find an HID Consumer device.")
+    def __init__(self, consumer_device=None):
+        """Create a ConsumerControl object that will send Consumer Control Device HID reports.
+        If consumer_device is None (the default), find a usb_hid device to use.
+        But an equivalent device can be supplied instead for other kinds of consumer devices,
+        such as BLE.
+        It only needs to implement ``send_report()``.
```

Comment:
How about just having a hid_devices parameter that can either be `usb_hid.devices` or `ble_hid.devices` rather than having one version look through a list and the other not?

I want to construct the devices on the ble side dynamically anyway and it's easy enough to put them in a list.